libdnf/repo/librepo_private.hpp

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Why call the header `librepo_private.hpp`? The header is private and we don't normally use the `_private` suffix, just call it `librepo.hpp`?

Reply:
Sometimes we used the suffix. By agreement, we will use the suffix only if there is a public header of the same name. Renaming to `librepo.hpp`.